packages/api-reference/src/components/Content/Schema/SchemaProperty.vue

```diff
@@ -310,15 +310,15 @@ const discriminatorType = discriminators.find((r) => {
       <div
         v-if="
           optimizedValue?.items &&
-          typeof discriminatorType === 'string' &&
+          typeof discriminator === 'string' &&
           typeof optimizedValue.items === 'object' &&
-          discriminatorType in optimizedValue.items &&
-          Array.isArray(optimizedValue.items[discriminatorType]) &&
+          discriminator in optimizedValue.items &&
+          Array.isArray(optimizedValue.items[discriminator]) &&
           level < 3
         "
         class="property-rule">
         <Schema
-          v-for="schema in optimizedValue.items[discriminatorType]"
+          v-for="schema in optimizedValue.items[discriminator]"
```

Comment:
this is the actual fix, everything else is prettier at work

Reply:
prettier or biome ™️